documentation/configuringDynawoAlgorithms/DynawoAlgorithmsInputFiles.tex

```diff
@@ -183,7 +185,7 @@ \subsubsection{MC mode: margin calculation}
 \textbf{The starting time defined in this .jobs file should be the end time of the load variation jobs file.}
 This situation should not contain any load variation model. 
 
-Two scenarios will be applied: `DisconnectLine' and DisconnectGroup'. 
+Two scenarios will be applied: 'DisconnectLine' and 'DisconnectGroup'. 
```

Comment:
I am not sure it is ok in latex like this I think the backtick is better. Did you compile the pdf in texmaker ?